unst_msh_gen/ocn_ww3.py

```diff
@@ -18,18 +20,21 @@
 
 from spacing import *
 
+# Load the configuration file
+config = configparser.ConfigParser()
+config.read('config.ini')
+
 # Create the parser
 parser = argparse.ArgumentParser(description="Run mesh gen with specific region settings.")
 
-# Add an argument for the black_sea option
-parser.add_argument('--black_sea', type=int, default=3,
+# Command-line arguments with defaults from config file
```

Comment:
This is a little confusing, can we just have either command line arguments or add this to the config file?

Reply:
yes, I added this to be dynamic, both parsing with CL or adding it to config will work. If you prefer to only having one of them, I can do that. Let me know which one is preferred?

Reply:
I think everything in the config file seems like a good idea. You can still set defaults and then over-ride with what is in the configuration if that's something you are interested in.  We've done that in other places, I could search for the example if needed.

Reply:
I saw that some of the instructions were updated to not have black sea option as an argument input, but I don't see this update here, is this the plan or something different?